projects/canopy/src/lib/utils/dom.service.ts

```diff
@@ -17,14 +17,22 @@ export class LgDomService {
    * @return The new value for the property
    *
    * @example
-   *     toggleIdInStringProperty('id-1 id-2', { id: 'id-1' }, { id: 'id-3' })
-   *     // returns 'id-1 id-2'
+   *     toggleIdInStringProperty('red blue', { id: 'red' }, { id: 'green' })
+   *     // returns 'blue green'
    */
   toggleIdInStringProperty(
     property = '',
     oldElement: { id: string },
     newElement?: { id: string },
   ): string {
+    if (!newElement && property === null) {
```

Comment:
`property` can't be `null` can it? Look like it is defaulted to `''` in the function param

Reply:
This works because it allows `null` to be passed through and is then evaluated in the `if` - it doesn't turn `null` into an empty string.I think it only defaults to a empty string if `property` is `undefined`.